apps/namada-interface/src/utils/theme/theme.ts

```diff
@@ -101,7 +101,10 @@ export type DesignConfiguration = {
   spacers: Spacers;
   borderRadius: BorderRadius;
   typography: Typography;
-  themeConfigurations: { isLightMode?: boolean; themeName: ThemeName };
+  themeConfigurations: {
```

Comment:
Just a thought, and I realized this was not introduced in this PR, so this comment can be ignored! But thinking about the naming convention here, it's usage requires:

`DesignConfiguration.themeConfigurations` and `DesignConfigurations.themeConfigurations.themeName`, which feels quite redundant and hard to read. I see this in a lot of places in our code, and feel like the following is adequately clear:

```typescript
export type DesignConfiguration = {
  ...
  theme: {
    colorMode: ColorMode;
    name: ThemeName;
  }
}
```
Instead of https://github.com/anoma/namada-interface/blob/360c09425f0799b7ffe67899d94e7f572d2cd1b5/apps/namada-interface/src/utils/theme/theme.ts#L99-L108

I think the hierarchical structure of this type makes it clear what each thing is. Like I said, not important to this PR, but just a thought I had! I'm curious what others think.

Reply:
I also realize it's a big hassle to change, now that we're loading the theme in styled-components props all over the place, so definitely not a concern for this PR!

Reply:
I introduced this some time ago for the purpose that we might need these constants to define exceptions in the use sites where we use the theme.

styled components has this root level `theme` object by default and we defined it's type in `/src/utils/theme/styled.d.ts` by `export interface DefaultTheme extends DesignConfiguration {}`.

I feel that the whole thing is theme, this is just part of the theme, so maybe a more specific name could be more descriptive. But I agree let's do this seperately, I added a note https://github.com/anoma/namada-interface/issues/3#issuecomment-1271288269.

so we use it like:
```ts
// now
const { colorMode } = theme.themeConfigurations;

// maybe one of these
const { colorMode } = theme.configurations;
const { colorMode } = theme.constants;
const { colorMode } = theme.parameters;
```